includes/sqp/1-4k-workflow.md

```diff
@@ -4,31 +4,18 @@
 
     **Depending what's released first and available the following Workflow Logic will be used:**
 
-    - When a (U)HD WEBDL is released it will download the (U)HD WEB-DL. (streaming services)
-    - When a (U)HD BHDStudio is released it will download the (U)HD BHDStudio releases.
-    - Where the UHD will upgrade the HD.
+    - When a 1080p WEBDL is released it will download the 1080p WEB-DL. (streaming services)
+    - When a BHDStudio is released it will download the BHDStudio releases.
+    - If no BHDStudio can be found (not existing or limited sources) it will download a 1080p Bluray Encodes (less or not streaming optimized).
+    - When a 2160p WEBDL is released it will download the 2160p WEB-DL. (streaming services)
+    - When a 2160p BHDStudio is released it will download the 2160p BHDStudio releases.
+    - Where the 2160p will upgrade the 1080p.
```

Comment:
Maybe change this to "1080p releases will be upgraded to 2160p releases when available."

Reply:
Changed.